code/centauri/ibc/modules/ics02_client.md

```diff
@@ -0,0 +1,365 @@
+## ICS02_CLIENT
+
+ICS02 defines the light client specification for the protocol.
+
+### Defining a light client
+
+To define a light client, there are a few things that must be available  
+- Proto file specifying client state, consensus state, header, misbehaviour and client message
+- Compile the proto files to rust
+- Define the rust equivalents of the structures in the compiled proto files
+- Define conversions between equivalent structs in compiled proto and manually defined
+- Implement `Protobuf` trait for these structs
+- Implement the `ClientState`, `ConsensusState` and `ClientMessage` for the appropriate structs
+- Implement `ClientDef`
+
+
+**Sample Proto file**
+```proto
+    syntax = "proto3";  
+
+    package ibc.lightclients.test.v1;  
+
+    import "google/protobuf/timestamp.proto";
+
+    message Authority {
+        // ed25519 public key of the authority
+        bytes public_key = 1;
+        // authority weight
+        uint64 weight = 2;
+    }
+
+    // ClientState
+    message ClientState {
+        // Latest chain height
+        uint32 latest_height = 1;
+
+        // current authority set id
+        uint64 current_set_id = 2;
```

Comment:
current_set_id in context of message ClientState could mean client_current_set_id . So it is not. But documented that current_set_id is current_authority_set_id. Why not too put authority into field name and no need for documentation?